gcp/appengine/frontend_handlers.py

```diff
@@ -300,7 +300,8 @@ def add_source_info(bug, response):
   response['source'] = source_repo.link + source_path
   response['source_link'] = response['source']
   if source_repo.human_link:
```

Comment:
Can we rename this to `human_link_template`?

Reply:
> Can we rename this to `human_link_template`?

I'm unsure? 

I can rename it at https://github.com/google/osv.dev/blob/5a8605df882d8a24c086655338e5f43fbddfe627/osv/models.py#L831 but I'm unclear of the ramifications on the existing data in Datastore of doing so, and how to handle that transition?

Alternatively, I could add a new property called `human_link_template` and adjust this PR to support both templated links and the previous behaviour?

Reply:
Ah good point, let's not rename this for now then, as long as people are aware that this is a template.